src/Microsoft.VisualStudio.LanguageServices.Razor/RazorSyntaxFactsService.cs

```diff
@@ -3,11 +3,12 @@
 
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis.Host;
 using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
-    public abstract class RazorSyntaxFactsService
+    public abstract class RazorSyntaxFactsService : ILanguageService
```

Comment:
My general comment here, I don't really want 'roslynize' the wrong API. For instance this is in the wrong namespace/assembly. I think the simplest thing is to copy these base classes